test/local/ip_profiling.js

```diff
@@ -49,13 +49,11 @@ function makeRoutes (options = {}, requireMocks) {
 
 function runTest(route, request, assertions) {
   return new P(function (resolve, reject) {
-    route.handler(request, function (response) {
-      if (response instanceof Error) {
-        reject(response)
-      } else {
-        resolve(response)
-      }
-    })
+    try {
+      return route.handler(request).then(resolve, reject)
```

Comment:
@deeptibaghel Similar to vlad's [comment](https://github.com/mozilla/fxa-auth-server/pull/2441#issuecomment-395619775), the `runTest` function needs to be tweaked a bit now that routes are async. This is an example that I think involves the least amount of changes.